src/app/@ansyn/menu-items/tools/components/image-processing-control/image-processing-control.component.ts

```diff
@@ -93,4 +98,27 @@ export class ImageProcessingControlComponent {
 	close() {
 		this.expand = false;
 	}
+
+	ngOnInit(): void {
+		this.resetParams();
+		this.store$.select(toolsStateSelector)
```

Comment:
i think you shouldn't create selector here... 
```typescript
  selectorName$ = this.store$.select(toolsStateSelector)
     .withLatestFrom(this.store$.select(mapStateSelector), this.store$.select(toolsStateSelector))
     .do(res => {
		if (res[0].imageProcessingHash[res[1].activeMapId] !== undefined) {
			this.params.forEach(param => {
				this.params[this.arrayObjectIndexOf(this.params, param.name, 'name')].value = res[0].imageProcessingHash[res[1].activeMapId][param.name];
			});
		}
	});
```

and in `ngOnInit`, `ngOnDestroy` just subscibe/unsubscibe:
```typescript
ngOnInit() {
  this.subscriptions.push(selectorName$.subscribe())
}
ngOnDestroy() {
 this.subscriptions.forEach(sub => sub.unsubsribe())
}

```